.github/workflows/nightly.yml

```diff
@@ -0,0 +1,133 @@
+# The last two jobs, unit testing and runnng TAU=full installation wait until the first finishes and re-uses the TAU=minimal install
+# Unfortunately, there is a lot of duplicated code that could be simplified/eliminated with YAML anchors and aliases,
+# but GitHub Actions does not yet support YAML anchors and aliases.
+# Clever use of global environment variables could help eliminate some of it, but the best options would be to just use aliases and anchors
+
+name: TAU nightly testing
+
+on:
+  schedule:
+    - cron: '18 10 * * *' # 10:18 UTC daily, 3:18 Pacific
+
+env:
+  __TAUCMDR_PROGRESS_BARS__: "disabled"
+  __TAUCMDR_SYSTEM_PREFIX__: "system"
+  CI: "ON"
+  INSTALLDIR: "taucmdr"
+  PIP_NO_CLEAN: "ON"
+  PIP_PREFER_BINARY: "ON"
+  TZ: "UTC"
+
+jobs:
+  Install-taucmdr-minimal:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-18.04, ubuntu-16.04]
+        python_version: ['2.7']
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 9999
+
+      - name: Install system packages
+        run: |
+          sudo apt-get update -qq || true
+          sudo apt-get install -y --no-install-recommends gcc g++ gfortran mpi-default-bin mpi-default-dev
+
+      # - name: Setup Python
+      #   uses: actions/setup-python@v1
+      #   with:
+      #     python-version: '3.6'
+      #     architecture: 'x64'
+
+      - name: System info
+        id: info
+        run: |
+          uname -a
+          mount
+          gcc --version
+          echo "github.event_name: ${{ github.event_name }}"
+          echo "github.ref:        ${{ github.ref }}"
+          echo "matrix.os:         ${{ matrix.os }}"
+          export INSTALLDIR="${HOME}/${INSTALLDIR}"
+          echo "::set-output name=install_dir::${INSTALLDIR}"
+          echo "::set-env name=INSTALLDIR::${INSTALLDIR}"
+          echo "::set-output name=home_dir::${HOME}"
+          echo "::set-output name=taucmdr_system::${INSTALLDIR}/${__TAUCMDR_SYSTEM_PREFIX__}"
+          echo "::set-env name=__TAUCMDR_SYSTEM_PREFIX__::${INSTALLDIR}/${__TAUCMDR_SYSTEM_PREFIX__}"
+
+# No caching available for cron jobs
```

Comment:
Also, there is less urgency with a nightly build, there is no parallel job logic here. We could add it, but the GitHub Actions timeout is 4 hours so I don't think we need it unless we start bumping up against that.